app/assets/javascripts/discourse/app/components/invite-link-panel.js

```diff
@@ -112,5 +108,15 @@ export default Component.extend({
         }
         model.setProperties({ saving: false, error: true });
       });
+  },
+
+  _setDefaultSelectedGroups() {
```

Comment:
If these are meant to be modified by plugin api they shouldn't be designated private using an underscore. I recommend removing the underscore.

Reply:
That makes sense, good recommendation.